.gitignore

```diff
@@ -1,3 +1,15 @@
+# generated UIs
```

Comment:
What is the problem in adding generated UIs here ?

Reply:
Travis complains about missing files then.